contrib/vmstat.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env drgn
+# Copyright (c) SUSE Linux.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""Dump /proc/vmstat statistics.
+
+   Skip the last enumerator item as it holds the number of enumrator values."""
+
+from drgn.helpers.linux.cpumask import for_each_online_cpu
+from drgn.helpers.linux.kconfig import get_kconfig
+from drgn.helpers.linux.percpu import per_cpu
+
+
+def print_event_line(event, counter):
+    print(f"{event.name:<36} {counter.value_():>16}")
+
+
+print(f'{"Event":<36} {"Count":>16}')
+
+# 1) vm_zone_stat statistics
+print('VM_ZONE_STAT:')
+vm_zone_stat = prog['vm_zone_stat']
+for event in prog.type('enum zone_stat_item').enumerators[:-1]:
+    print_event_line(event, vm_zone_stat[event.value].counter)
+print()
+
+# 2) vm_node_stat statistics
+print('VM_NODE_STAT:')
+vm_node_stat = prog['vm_node_stat']
+for event in prog.type('enum node_stat_item').enumerators[:-1]:
+    print_event_line(event, vm_node_stat[event.value].counter)
+print()
+
+# 3) vm_numa_event statistics are available only if CONFIG_NUMA is enabled
+if 'CONFIG_NUMA' in get_kconfig(prog):
+    print('VM_NUMA_EVENT:')
+    vm_numa_event = prog['vm_numa_event']
```

Comment:
This doesn't work before this commit in 5.14: https://github.com/torvalds/linux/commit/f19298b9516c1a031b34b4147773457e3efe743b

Since this is a `contrib` script, I'll leave it up to you whether you want to handle older kernels.

Reply:
Sure, let's make check if `vm_numa_event` is available or not.